<?xml version="1.0"?>

<launch>

    <arg name="gdb" default="false"/>
    <arg if="$(arg gdb)" name="launch-prefix" value="gdb -ex run --args"/>
    <arg unless="$(arg gdb)" name="launch-prefix" value=""/>

    <node name="rgbd_encoder" pkg="rgbd" type="rgbd_server" output="screen" launch-prefix="$(arg launch-prefix)">
        <remap from="cam_info" to="/camera/rgb/camera_info"/>
        <remap from="rgb_image" to="/camera/rgb/image_rect_color"/>
        <remap from="depth_image" to="/camera/depth_registered/hw_registered/image_rect_raw"/>
        <remap from="output" to="/camera/rgbd"/>
    </node>

</launch>
